### PR TITLE
Issue 188: Remove and update menuitems and menu mappings

### LINKED ISSUE #188: Remove/update menuitems and menu
remove `menuitem`s as they are not part of the HTML living standard:
- [x] menuitem type=command 
- [x] menuitem type=checkbox
- [x] menuitem type=radio

`menu` *is* part of the HTML living standard, so this should stay in AAM, but it does not accept a `type` attribute.
- [x] revise menu to remove reference to type

### PR DESCRIPTION
**Pull request by scottaohara**
Closes #188 

removed element mappings for:
- a with parent menu
- button with parent menu
- input type=checkbox with parent menu
- input type=image with parent menu
- input type=radio with parent menu
- menuitem type=checkbox
- menuitem type=command
- menuitem type=radio

removed attribute mappings for:
- checked referencing menuitem types
- disabled referencing menuitem
- icon for menuitem element
- label referencing menu and menuitem elements
- radiogroup for menuitem element
- title for menuitem element

menu updated to map to role=list
  - add comment for info on role=menu to see ARIA

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/pull/214.html" title="Last updated on Sep 11, 2019, 1:29 PM UTC (6056938)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/214/24f3ea1...6056938.html" title="Last updated on Sep 11, 2019, 1:29 PM UTC (6056938)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2412,40 +2337,17 @@ <h3>HTML Element Role Mappings</h3>
...
+              <th>
+                <a data-cite="HTML">`menu`</a>
+              </th>
+              <td class="aria"><a class="core-mapping" href="#role-map-list">`list`</a> role </td>
+              <td class="ia2"><div class="general">Use WAI-ARIA mapping</div></td>
+              <td class="uia"><div class="general">Use WAI-ARIA mapping</div></td>
+              <td class="atk"><div class="general">Use WAI-ARIA mapping</div></td>
+              <td class="ax"><div class="general">Use WAI-ARIA mapping</div></td>
+              <td class="comments">
+                The <a data-cite="HTML">`menu`</a> element is a semantic alternative to the <a data-cite="HTML">`ul`</a> element, and has no implemented mappings or behavior similar to ARIA's <a class="core-mapping" href="#role-map-menu">`role="menu"`</a>.
```
Comment (stevefaulkner) [inline index.html]: suggest rewording "has no implemented mappings or behavior similar to ARIA's <a class="core-mapping" href="#role-map-menu">`role="menu"`</a>"

to "has no implemented mappings or behavior that reflect the semantics of the ARIA <a class="core-mapping" href="#role-map-menu">`menu`</a> role.

**Review by stevefaulkner — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (scottaohara) [reply index.html]: updated

**Review by stevefaulkner — APPROVED** [review]

**Comment by scottaohara** [comment]
Added "not ready for merge" label as there are some issues that need to be addressed outside of this PR before merging.

**Comment by scottaohara** [comment]
Adding todo item of adding a link in the comment section for `menu` to point to HTML's obsolete section.

- [x] add https://html.spec.whatwg.org/multipage/obsolete.html#attr-menu-type

**Comment by scottaohara** [comment]
latest push completes todo to reference/link to obsolete `menuitem` and `menu type` in HTML spec.  changelog date updated.  this will also close #237 which is duplicate to this PR.

**Review by stevefaulkner — APPROVED** [review]
